src/astro/sql/operators/load_file.py

```diff
@@ -80,13 +83,23 @@ def load_data_to_table(self, input_file: File) -> Table:
         for file in get_files(
             input_file.path, input_file.conn_id, normalize_config=self.normalize_config
         ):
-            database.load_pandas_dataframe_to_table(
-                source_dataframe=file.export_to_dataframe(),
-                target_table=self.output_table,
-                if_exists=if_exists,
-                chunk_size=self.chunk_size,
-            )
-            if_exists = "append"
+            if self.use_native_support and database.check_optimised_path(
+                source_file=file, target_table=self.output_table
+            ):
+                database.optimised_transfer(
+                    source_file=file,
+                    target_table=self.output_table,
+                    if_exists=self.if_exists,
+                    **self.kwargs,
+                )
+            else:
+                database.load_pandas_dataframe_to_table(
+                    source_dataframe=file.export_to_dataframe(),
+                    target_table=self.output_table,
+                    if_exists=if_exists,
+                    chunk_size=self.chunk_size,
+                )
+                if_exists = "append"
```

Comment:
It would be great if we kept the operator as simple as possible and handled optimised paths from within the database class, as part of the `load_file_to_table` method